ep/build-velox/src/build_velox.sh

```diff
@@ -9,8 +9,8 @@ COMPILE_VELOX=OFF
 ENABLE_EP_CACHE=OFF
 VELOX_HOME=/root/velox
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/jinchengchenghh/velox.git
+VELOX_BRANCH=ifthen
```

Comment:
set back to oap-project/velox

Reply:
wait for this https://github.com/oap-project/velox/pull/71

Reply:
switch to https://github.com/oap-project/velox/pull/74